acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -0,0 +1,930 @@
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  \n",
+    "See the License for the specific language governing permissions and  \n",
+    "limitations under the License.\n",
+    "\n",
+    "# Benchmark TensorRT models inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial shows how to convert a bundle in the MONAI model zoo to a TensorRT engine-based TorchScript for better performance and benchmark the difference between the PyTorch module and the TensorRT engine-based TorchScript.  \n",
+    "The `spleen_ct_segmentation` bundle is used as the example in this tutorial. The inference latency before and after the covert process will be compared. Feel free to modify the `spleen_ct_segmentation` bundle to other bundles in the MONAI model zoo which support TensorRT conversion. The TensorRT conversion is supported since **MONAI 1.2** with **Torch-TensorRT 1.4.0** and **TensorRT 8.5.3**. Please use the MONAI Docker whose version is greater than 1.2 or ensure the environment meets the min version requirement. The supported bundle names are listed below. Please modify the net_id and input_shape in the tutorial to fit different bundles.\n",
+    "\n",
+    "```\n",
+    "spleen_ct_segmentation\n",
+    "endoscopic_tool_segmentation\n",
+    "endoscopic_inbody_classification\n",
+    "pathology_tumor_detection\n",
+    "```\n",
+    "\n",
+    "The Spleen dataset can be downloaded from http://medicaldecathlon.com/.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -c \"import monai\" || pip install -q \"monai-weekly[nibabel, tqdm]\"\n",
+    "!python -c \"import matplotlib\" || pip install -q matplotlib\n",
+    "!python -c \"import pandas\" || pip install -q pandas\n",
+    "!python -c \"import torch_tensorrt\" || pip install torch_tensorrt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import tempfile\n",
+    "\n",
+    "import torch\n",
+    "import torch_tensorrt\n",
+    "import numpy as np\n",
+    "from ignite.engine import Engine\n",
+    "import pandas as pd\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import monai\n",
+    "from monai.apps import download_and_extract\n",
+    "from monai.config import print_config\n",
+    "from ignite.engine import Events\n",
+    "from collections import OrderedDict\n",
+    "from monai.engines import IterationEvents\n",
+    "\n",
+    "print(f\"Torch-TensorRT version: {torch_tensorrt.__version__}.\")\n",
```

Comment:
Hi @wyli ,

Do you think it's better to include `torch-trt` as part of MONAI optional dependency?
Currently, the main concern is that TensorRT environment is complicated and related to CUDA, cuDNN, TensorRT lib, etc. `pip install torch-trt` may not work well.

Thanks.

Reply:
FYI, if directly run the tutorial in the ci/ci environment, it reports an torch-tensorrt related error, which means the torch-tensorrt directly installed by pip doesn't work in the environment.

Thanks,
Bin